src/frontend/src/pages/myfeeds.js

```diff
@@ -142,14 +115,16 @@ export default function MyFeeds() {
     }
   }
 
-  function handleBlur(event, ref) {
-    ref.current.validate(event.target.value, true);
+  function deletionCallback(id) {
+    const updatedHash = { ...feedHash };
+    delete updatedHash[id];
+    updateFeedHash(updatedHash);
   }
 
   return userInfo.id ? (
     <PageBase title="My Feeds">
       <div className={classes.margin}>
-        <ValidatorForm onSubmit={addFeed}>
+        <ValidatorForm onSubmit={addFeed} instantValidate={false}>
```

Comment:
I think @humphd commented on this in my current PR it should be `onSubmit={() => addFeed()}`

Reply:
Good to know, thanks. I saw that you already made this change in your PR. When I'm fixing the merge conflicts between our work, I'll favour yours, in this situation.